site/guide/organize-model-inventory.qmd

```diff
@@ -0,0 +1,28 @@
+---
+title: "Organize models in the inventory"
```

Comment:
Is it" organize". "customize", or "configure" the model inventory? I feel like the last one is perhaps closest to what's described in the steps.